app/scripts/controllers/buildConfig.js

```diff
@@ -207,7 +211,7 @@ angular.module('openshiftConsole')
 
         // Sort now to avoid sorting on every digest loop.
         $scope.orderedBuilds = BuildsService.sortBuilds($scope.builds, true);
-        $scope.latestBuild = $scope.orderedBuilds.length ? $scope.orderedBuilds[0] : null;
+        $scope.latestBuild = latestBuild($scope.orderedBuilds);
```

Comment:
i think we would be fine just doing `_.first($scope.orderedBuilds)` here and dropping the added method.  we dont need it to be specifically set to null, just some falsey / non-existent value.